### PR TITLE
Add capability to remove existing answers

### PR DESCRIPTION
**Pull request by fkm**

### CODE REVIEW

File: addon/lib/field.js [hunk 1]
```diff
@@ -178,20 +180,32 @@ export default EmberObject.extend(Evented, {
...
-        mutation: this.get(`saveDocument${type}Mutation`),
+    if (value === null || value.length === 0) {
+      return yield this.apollo.mutate({
+        mutation: removeAnswerMutation,
         variables: {
           input: {
-            question: this.get("question.slug"),
-            document: this.get("document.id"),
-            value: this.get("answer.value")
+            answer: atob(this.get("answer.id")).split(":")[1]
```
Comment (czosel) [inline addon/lib/field.js]: Use `import { atob } from "ember-caluma/helpers/atob";` instead

File: docker-compose.yml [hunk 2]
```diff
@@ -34,4 +34,4 @@ services:
 
 volumes:
   dbdata:
-  minio_data:
\ No newline at end of file
+  minio_data:
```
Comment (czosel) [inline docker-compose.yml]: Can you configure your editor to stop doing that? ^^

File: tests/integration/components/cf-field-test.js [hunk 3]
```diff
@@ -46,6 +46,16 @@ module("Integration | Component | cf-field", function(hooks) {
...
   });
 
+  test("it allows deleting existing input", async function(assert) {
+    assert.expect(1);
+
+    await render(hbs`{{cf-field field=field}}`);
+    this.set("field.answer.value", "Test");
+
+    await fillIn("input", "");
+    assert.equal(this.field.answer.value, "", "Value was removed.");
```
Comment (czosel) [inline tests/integration/components/cf-field-test.js]: Shouldn't the whole answer be removed?

File: docker-compose.yml (hunk 2 above)
Comment (fkm) [reply docker-compose.yml]: Newlines at EOF?

File: addon/lib/field.js (hunk 1 above)
Comment (fkm) [reply addon/lib/field.js]: Ok

What's the reason behind the helper?

File: tests/integration/components/cf-field-test.js (hunk 3 above)
Comment (fkm) [reply tests/integration/components/cf-field-test.js]: Good point! I'll look into that. Probably need to add Mirage to the test.

File: docker-compose.yml (hunk 2 above)
Comment (czosel) [reply docker-compose.yml]: yep

File: addon/lib/field.js (hunk 1 above)
Comment (czosel) [reply addon/lib/field.js]: The helper just exists to avoid having the same logic spread across many files.

File: docker-compose.yml (hunk 2 above)
Comment (fkm) [reply docker-compose.yml]: That's something I mostly do manually. But yeah. I can stop.

What's the reasoning behind not having newlines at the end of the files for some file types? And what types require newlines and which don't?

Comment (czosel) [reply docker-compose.yml]: Ah ok. I don't know why it's preferred not to have those newlines, but I do know that it's nice to have a consistent default for this all over the repository. My editor is configured to remove those newlines, and I think most of the people in our team handle it the same way.
I'd just like to get of the extra noise in those merge requests :wink:

Comment (fkm) [reply docker-compose.yml]: Ok

File: tests/integration/components/cf-field-test.js (hunk 3 above)
Comment (fkm) [reply tests/integration/components/cf-field-test.js]: I don't really see a way to create the fields from (Mirage)GraphQL without the cf-form component. I'll have another look at it later. This is probably going to take a moment.

Comment (czosel) [reply tests/integration/components/cf-field-test.js]: Wanna create an issue and move on for the moment?

Comment (fkm) [reply tests/integration/components/cf-field-test.js]: Maybe the best solution to get stuff done quickly.

Comment (czosel) [reply tests/integration/components/cf-field-test.js]: Alright, please don't forget to create the issue :wink:

Comment (fkm) [reply tests/integration/components/cf-field-test.js]: https://github.com/projectcaluma/ember-caluma/issues/181

**Pull request merged**